src/main/resources/META-INF/resources/primefaces/locales/locales.js

```diff
@@ -0,0 +1,1363 @@
+    dayNamesShort: ['Ne', 'Po', 'Ut', 'Sr', 'Če', 'Pe', 'Su'],
+    dayNamesMin: ['N', 'P', 'U', 'S ', 'Č', 'P ', 'S'],
+    weekHeader: 'Tjedan',
+    firstDay: 1,
+    isRTL: false,
+    showMonthAfterYear: false,
+    yearSuffix: '',
+    timeOnlyTitle: 'Samo vrijeme',
+    timeText: 'Vrijeme',
+    hourText: 'Sat',
+    minuteText: 'Minuta',
+    secondText: 'Sekunda',
+    currentText: 'Sada',
+    ampm: false,
+    month: 'Mjesec',
+    week: 'Tjedan',
+    day: 'Dan',
+    allDayText: 'Cijeli dan'
+};
+
+/** Czech */
+PrimeFaces.locales['cs_CZ'] = {
+    closeText: 'Zavřít',
+    prevText: 'Předchozí',
+    nextText: 'Následující',
+    monthNames: ['Leden', 'Únor', 'Březen', 'Duben', 'Květen', 'Červen', 'Červenec', 'Srpen', 'Září', 'Říjen', 'Listopad', 'Prosinec'],
+    monthNamesShort: ['Led', 'Úno', 'Bře', 'Dub', 'Kvě', 'Čer', 'Čvc', 'Srp', 'Zář', 'Říj', 'Lis', 'Pro'],
+    dayNames: ['Neděle', 'Pondělí', 'Úterý', 'Středa', 'Čtvrtek', 'Pátek', 'Sobota'],
+    dayNamesShort: ['Ne', 'Po', 'Út', 'St', 'Čt', 'Pá', 'So'],
+    dayNamesMin: ['Ne', 'Po', 'Út', 'St', 'Čt', 'Pá', 'So'],
+    weekHeader: 'Týden',
+    firstDay: 1,
+    isRTL: false,
+    showMonthAfterYear: false,
+    yearSuffix: '',
+    timeOnlyTitle: 'Pouze čas',
+    timeText: 'Čas',
+    hourText: 'Hodina',
+    minuteText: 'Minuta',
+    secondText: 'Sekunda',
+    currentText: 'Nyní',
+    ampm: false,
+    month: 'Měsíc',
+    week: 'Týden',
+    day: 'Den',
+    allDayText: 'Celý den'
+};
+PrimeFaces.locales['cs'] = PrimeFaces.locales['cs_CZ'];
+
+/** Dutch */
+PrimeFaces.locales['nl'] = {
+    closeText: 'Sluiten',
+    prevText: 'Terug',
+    nextText: 'Volgende',
+    currentText: 'Huidig',
+    monthNames: ['Januari', 'Februari', 'Maart', 'April', 'Mei', 'Juni', 'Juli', 'Augustus', 'September', 'Oktober', 'November', 'December'],
+    monthNamesShort: ['jan', 'feb', 'mrt', 'apr', 'mei', 'jun', 'jul', 'aug', 'sep', 'okt', 'nov', 'dec'],
+    dayNames: ['Zondag', 'Maandag', 'Dinsdag', 'Woensdag', 'Donderdag', 'Vrijdag', 'Zaterdag'],
+    dayNamesShort: ['zo', 'ma', 'di', 'woe', 'do', 'vr', 'za'],
+    dayNamesMin: ['Zo', 'Ma', 'Di', 'Wo ', 'Do', 'Vr ', 'Za'],
+    weekHeader: 'Week',
+    firstDay: 1,
+    isRTL: false,
+    showMonthAfterYear: false,
+    yearSuffix: '',
+    timeOnlyTitle: 'Alleen tijd',
+    timeText: 'Tijd',
+    hourText: 'Uur',
+    minuteText: 'Minuut',
+    secondText: 'Seconde',
+    ampm: false,
+    month: 'Maand',
+    week: 'week',
+    day: 'Dag',
+    allDayText: 'Alle Dagen'
+};
+
+/** French */
+PrimeFaces.locales['fr'] = {
+    closeText: "Fermer",
+    prevText: "Précédent",
+    nextText: "Suivant",
+    monthNames: ["Janvier", "Février", "Mars", "Avril", "Mai", "Juin", "Juillet", "Août", "Septembre", "Octobre", "Novembre", "Décembre"],
+    monthNamesShort: ["Jan", "Fév", "Mar", "Avr", "Mai", "Jun", "Jul", "Aoû", "Sep", "Oct", "Nov", "Déc"],
```

Comment:
A few typos here: https://fr.wikipedia.org/wiki/Mois#Abr%C3%A9viations (i guess it's always been there)